core/arch/arm/kernel/sub.mk

```diff
@@ -5,8 +5,6 @@ srcs-$(CFG_EARLY_TA) += early_ta.c
 srcs-$(CFG_SECSTOR_TA) += secstor_ta.c
 endif
 srcs-y += pseudo_ta.c
-srcs-y += elf_load.c
-srcs-$(CFG_TA_DYNLINK) += elf_load_dyn.c
```

Comment:
`CFG_TA_DYNLINK ?= y` should be removed from `mk/config.mk`. Also mention in the commit text that loading dynamically linked TAs is always supported / cannot be disabled.

Reply:
I'll fix